contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -880,6 +880,8 @@ def _transcribe(item, parents, edit_rate, indent=0):
                 result.kind = otio.schema.TrackKind.Video
             elif media_kind in ("SoundMasterTrack", "Sound"):
                 result.kind = otio.schema.TrackKind.Audio
+            elif media_kind == "Descriptive Metadata":
+                result.kind = otio.schema.TrackKind.Data
             else:
```

Comment:
`Descriptive Metadata` is not the media kind for Data tracks your referring too.  The media kind for the data track is `DataEssenceTrack`.